### PR TITLE
Adding 'displayBlock' prop to FloatingDropdown component

### PR DESCRIPTION
**Pull request by barboaron**
# ✨ Pull Request

### 💁 Description

...

### 💭 Motivation

...

### ☝️ TODOs 

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_559/

File: src/components/FloatingDropdown/docs/index.story.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ export default {
             { title: 'MaxWidth', source: examples.maxWidth },
             { title: 'Disabled', source: examples.disabled },
             { title: 'Ellipsis', source: examples.ellipsis },
+            { title: 'displayBlock', source: examples.displayBlock },
```
Comment (jonathanadler) [inline src/components/FloatingDropdown/docs/index.story.tsx:68]:
```suggestion
            { title: 'Block Display', source: examples.displayBlock },
```

File: src/components/FloatingDropdown/FloatingDropdownBase.st.css [hunk 2]
```diff
@@ -68,6 +68,14 @@
...
 }
 
+.displayBlockBtn {
+    width: 100%;
+}
+
+.displayBlockContent {
+    justify-content: space-between;
+}
+
```
Comment (jonathanadler) [inline src/components/FloatingDropdown/FloatingDropdownBase.st.css:78]: Really not critical, 
but I think I would have just added one class to the root component, and then use that to target the button and the content.
Something like:
```css
.displayBlock .btn {
  width: 100%;
}

.displayBlock .content {
  justify-content: space-between;
}
```

But again, this isn't critical

**Review by jonathanadler — COMMENTED** [review]
Great work!

I left some minor comments.

I'll ask @michalvardi to give a design review, and once we get an ok, we can merge 👍 

Thanks!

**Review by jonathanadler — APPROVED** [review]

**Pull request merged**